packages/wdio-reporter/README.md

```diff
@@ -131,10 +131,11 @@ These events are containing data about the test regardless of the framework it i
 SuiteStats {
   type: 'suite',
   start: '2018-02-09T13:30:40.177Z',
-  duration: 0,
+  _duration: 0,
```

Comment:
@christian-bromann Why `duration` named with underscore in reporter code? It will cause errors in most linters...

Reply:
See https://github.com/webdriverio/v5/blob/7d3ad6dd4ecc4c860f01bf53ceffd62a86c5b200/packages/wdio-reporter/src/stats/runnable.js#L9

Reply:
@BorisOsipov because there is a `duration` getter which returns the duration based on whether the test has ended yet or not.

> It will cause errors in most linters...

IMO if it doesn't cause errors in our linter, it should be fine.

Reply:
having that said, I think this change and all the following regarding this should be reverted

Reply:
ok